docs/rfcs/XeGPU.md

```diff
@@ -701,6 +700,332 @@ An example on how to perform transpose using load with chunk_size in SIMT flavor
 
 ```
 
+## Workgroup level XeGPU Operations
+
+By allowing XeGPU operating on workgroup level data size, it provides a concise IR for tensor compiler instead of multiple level nested loop IR for subgroup and work item level operation. To enable XeGPU operate the workgroup level, we introduce `wg_map` attribute to specify how the data is distributed across subgroups. `wg_map` enables tensor compiler to express the cooperative operation among subgroups by specifying a `wg_map` to partition data among subgroups without modifying the IR representation other required when using loop nest IR. The attribute allows tensor compiler to control the block size for both the workgroup and subgroup and perform autotuning as the number of subgroups, layout, and tensor size per subgroups are critical performance knobs.
+
+**Attribute xegpu.wg_map**
+
+`wg_map` specifies how a n-d tensor (defined by the tensor descriptor) is partitioned among subgroup within a workgroup. wg_map consists of three parameters:
```

Comment:
Why not merge have wg_map include the full layout down to wi?
`wg_map = sg_layout, wi_layout, wi_data`.
By having two layouts, you allow nested wrapping around (you can wrap work-items around the subgroup tile, and also wrap the subgroups around the entire tensor). Also easier for compiler to reason about and deal with single layout.

Reply:
We took the feedback. This version just expand sg_map to include sg_layout.